src/main/java/com/google/sps/data/Course.java

```diff
@@ -20,42 +20,43 @@
  * A course class. Better documentation to be added later.
  */
 public class Course {
-    private String name;
-    private String courseID;
-    private String subject;
-    private boolean isRequired;
-    private List<Section> sections;
-
-    public Course(String name, String courseID, String subject, boolean isRequired, List<Section> sections) {
-        this.name = name;
-        this.courseID = courseID;
-        this.subject = subject;
-        this.isRequired = isRequired;
-        this.sections = sections;
-    }
-
-    public String getName() {
-        return this.name;
-    }
-
-    public String getCourseID() {
-        return this.courseID;
-    }
-
-    public String getSubject() {
-        return this.subject;
-    }
-
-    public boolean getIsRequired() {
-        return this.isRequired;
-    }
-
-    public boolean isIsRequired() {
-        return this.isRequired;
-    }
-
-    public List<Section> getSections() {
-        return this.sections;
-    }
-
-}
\ No newline at end of file
+  private String name;
+  private String courseID;
+  private String subject;
+  private float credits;
+  private boolean isRequired;
+  private List<Section> sections;
+
+  public Course(String name, String courseID, String subject, float credits, boolean isRequired, List<Section> sections) {
+    this.name = name;
+    this.courseID = courseID;
+    this.subject = subject;
+    this.credits = credits;
+    this.isRequired = isRequired;
+    this.sections = sections;
+  }
+
+  public String getName() {
+    return this.name;
+  }
+
+  public String getCourseID() {
+    return this.courseID;
+  }
+
+  public String getSubject() {
+    return this.subject;
+  }
+
+  public float getCredits() {
+    return this.credits;
+  }
+
+  public boolean getIsRequired() {
```

Comment:
Getters for booleans generally drop the `get` so it reads easier: `isRequired()`